tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -233,6 +233,7 @@ describe('Test resource behavior after deleting GitRepo using keepResources opti
   )
 });
 
+if (Cypress.env('rancher_version') != '2.7') {
```

Comment:
I'm afraid this is not enough and the condition will be never met as `rancher_version` can be either `latest/devel/2.7` or `latest/2.7.x` so we'd need a regexp here.

Anyway it's not using `rancher_version` from inputs but in the end value from `steps.component.outputs.rm_version`  (see [here](https://github.com/rancher/fleet-e2e/blob/c01c8d1a433e2fe7fe3245bddd40ff7f8dc87fcd/.github/workflows/master-e2e.yaml#L273C5-L273C69)) which returns rancher container image like:
```
stgregistry.suse.com/rancher/rancher:v2.7-head
docker.io/rancher/rancher:v2.8-head
rancher/rancher:v2.7.10
...
```

I wonder why it is like that but I guess we should first fix it to report valid `rancher_version` instead of `rm_version`.

Reply:
This is resolved in later commit

Reply:
Thanks @thehejik for the input here.
It was a nice solution but hard one.
I will merge it now to skip errors on 2.7 and we can figure it out later an easier solution.

Thanks again

Reply:
Ah, forget it. I cannot merge it. :)